docs/concepts/solutions.rst

```diff
@@ -0,0 +1,91 @@
+.. _solutions_sdk:
+
+=========
+Solutions
```

Comment:
I think this should be `Samples`; if my problem is a CSP and the sampler returns an excited state then calling it a `solution` is pretty misleading

Reply:
At the concept level I especially used "solutions" for new users that won't know what "samples" mean rather than try to be precise.